client/components/order.js

```diff
@@ -13,17 +13,47 @@ export class Order extends Component {
 
   render() {
     const productName = this.props.currentOrder.products
-    const name =
+    var quantities = []
+    for (let i = 1; i <= 5; i++) {
```

Comment:
Is there a specific reason that this is based on i<= 5? shouldn't it be based on the stock of the item in the db?

Reply:
It is just a trial run to display a dropdown. Since we can't add to the cart two pairs of mittens of the same item yet, it doesn't make any sense to attach i to anything, once we have a proper functionality, I will change it.

Reply:
What I meant by that, is I can't grab eager loading (item-in-orders) table for some reason, probably because of dashes (I will show tomorrow). So, I'm just using whatever I can grab to make it more-less functional (but it's not how it suppose to be). So, instead of breaking my head, I prepared this functionality I guess, that we can easily edit it tomorrow, as soon as we figure out the main issue.

Reply:
and yes, i is going to be based on item stock